Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -42,6 +42,8 @@ public async Task InitDefaultCustomCommands()
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<GetRoles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-add\s*warn\s*-user\s*(?<User><@\d+>)\s*-reason\s*(?<Reason>.+)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(-user\s*(?<User><@&?\d+>))?\s*", serverId),
```

Comment:
@Marcin99b users nie ma sensu chyba że był w regexie opcjonalny bo opcjonalność już załatwia normalny template komendy?
```suggestion
                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.WarnsCommand", @"-warns\s*(?<User><@\d+>)", serverId),
```
z jakiegoś powodu był ten "&?"?

Reply:
`z jakiegoś powodu był ten "&?"?`

tak, bo czasami discord tak to formatuje 
nie wiem dokładnie od czego to zależy 
a w dokumentacji discorda było info że obie wersje są poprawne
nie widziałem info kiedy dokładnie pojawia sie wersja z &

Reply:
wiem, że & pojawia się przy rolach, ale nie wiedziałem, że przy user też
tylko to rodzi taki problem, że będzie w pole User łapać również wzmianki do roli
ale jeśli tak musi być to niech zostanie

Reply:
czyli coś mam zmienić czy jest ok?

Reply:
no powinno być to łapanie wersji z & - nawet dla zabezpieczenia 
a nawet mogłoby być `<\D{1,2}\d+>`
(\D to znak inny niż cyfra)

co sądzisz @gagyn

Reply:
niech będzie

Reply:
czyli za \D można wstawić cokolwiek innego jeśli nie jest cyfrą, czyli nie musi być ani & ani @?

Reply:
tak

Reply:
dlaczego jest okej wstawienie jakichkolwiek znaków w miejsce gdzie musimy wybrać usera, skoro powinno być @ albo @&?

Reply:
bo co jeśli discord coś zmieni u siebie
nie powinni
ale może wyjść jakaś aktualizacja 

dla nas ważne jest żeby złapać ID